src/API/useFetch.tsx

```diff
@@ -1,20 +1,27 @@
 //@ts-nocheck
-import React, { useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import axios from "axios";
 const GITHUB_BASE_URL = 'https://api.github.com';
 
+export interface repositoryCommitMessage  {
+  date: {}
```

Comment:
Why is `date` an object? Normally I see it as a string. Just wondering your thought process here. When using TY you should type the keys inside the object and there values. 

```
export interface repositoryCommitMessage  {
  date: {
            value: string;
            otherValue: string;
            }
```